e2e-tests/cypress/tests/integration/channels/channel/channel_bookmarks_spec.ts

```diff
@@ -174,6 +198,30 @@ describe('Channel Bookmarks', () => {
         // * Verify bookmark deleted
         cy.findByRole('link', {name: displayName}).should('not.exist');
     });
+
+    it('reorder bookmark', () => {
+        const {displayName: name1} = createFileBookmark({file: 'm4a-audio-file.m4a', displayName: 'custom displayname 1'});
+        const {displayName: name2} = createFileBookmark({file: 'm4a-audio-file.m4a', displayName: 'custom displayname 2'});
+
+        // # Start reorder bookmark flow
+        cy.findByTestId('channel-bookmarks-container').within(() => {
+            cy.findAllByRole('link').should('be.visible').as('bookmarks');
+            cy.get('@bookmarks').eq(-1).scrollIntoView();
+            cy.get('@bookmarks').eq(-2).should('contain', name1);
+            cy.get('@bookmarks').eq(-1).should('contain', name2);
+
+            // # Perform drag using keyboard
+            cy.get(`a:contains(${name1})`).
+                trigger('keydown', {keyCode: SpaceKeyCode}).
+                trigger('keydown', {keyCode: RightArrowKeyCode, force: true}).wait(TIMEOUTS.THREE_SEC).
+                trigger('keydown', {keyCode: SpaceKeyCode, force: true}).wait(TIMEOUTS.THREE_SEC);
```

Comment:
I did try that approach, but ran into blockers. Found some alternatives but then found we had this already in https://github.com/mattermost/mattermost/blob/4d2342ab8c3ff8b6d660cc6a721983e807db94b5/e2e-tests/cypress/tests/integration/channels/channel_sidebar/drag_and_drop_spec.ts#L37C1-L38C1

Reply:
I'd be interested in knowing what those blockers were, but this is fine. I was mostly just curious here since I haven't worked with this library myself